public/lesswrong-editor/package.json

```diff
@@ -64,6 +64,10 @@
     "@ckeditor/ckeditor5-track-changes": "^31.0.0",
     "@ckeditor/ckeditor5-ui": "^31.0.0",
     "@ckeditor/ckeditor5-watchdog": "^31.0.0",
+    "@types/ckeditor__ckeditor5-core": "33.0.3",
```

Comment:
@b0b3rt Is this right?

Reply:
We should remove those by running `yarn remove` for all of those that were added